### PR TITLE
Convert Context Menu to TypeScript

### PR DESCRIPTION
**Pull request by t3chguy**

### CODE REVIEW

**Comment by t3chguy** [comment]
Blocking on https://github.com/matrix-org/matrix-react-sdk/pull/4896 to simplify the conflicts

File: src/accessibility/context_menu/ContextMenuButton.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2015, 2016 OpenMarket Ltd
+Copyright 2018 New Vector Ltd
+Copyright 2019 The Matrix.org Foundation C.I.C.
```
Comment (jryans) [inline src/accessibility/context_menu/ContextMenuButton.tsx:4]: Are these the right copyright headers here and for all other new files?

Comment (t3chguy) [reply src/accessibility/context_menu/ContextMenuButton.tsx:4]: Well the code is existing just moved and without tracing through exactly which copyright was used when it was created I thought this was safest

Comment (jryans) [reply src/accessibility/context_menu/ContextMenuButton.tsx:4]: Ah right I see now, seems fine then.

**Pull request merged**